static/switch-btn.js

```
@@ -4,49 +4,56 @@ var switchButton = document.querySelector('.switch-button');
 var st4 = document.getElementsByClassName('st4');
 var st5 = document.getElementsByClassName('st5');
 var st6 = document.getElementsByClassName('st6');
-function changeMachine(color3,color4,color5,color6){
+
+function changeMachine(color3, color4, color5, color6) {
 	for (var i = 0; i < st3.length; i++) {
 		st3[i].style.fill = color3;
 	}
+
 	for (var i = 0; i < st4.length; i++) {
 		st4[i].style.fill = color4;
```

Biome (1.9.4)

[error] Shouldn't redeclare 'i'. Consider to delete it or rename it.

'i' is defined here:


(lint/suspicious/noRedeclare)